### DISCUSSION TITLE
how to generate `oamdev/kube-webhook-certgen` image

### DISCUSSION THREAD

**Discussion by huiwq1990** [kubevela/kubevela#4490] [Q&A] [answered]
I just found the image `oamdev/kube-webhook-certgen`, and how to generate it? And, what's the difference with `jet/kube-webhook-certgen`?

**Comment by wonderflow** [comment, marked as answer]
Please refer to https://github.com/oam-dev/kube-webhook-certgen

**Reply by huiwq1990** [reply]
thx
